pkg/skaffold/build/custom/dependencies.go

```diff
@@ -26,15 +26,24 @@ import (
 )
 
 // GetDependencies returns dependencies listed for a custom artifact
-func GetDependencies(ctx context.Context, workspace string, a *latest.CustomArtifact) ([]string, error) {
-	files, err := docker.WalkWorkspace(workspace, a.Dependencies.Ignore, a.Dependencies.Paths)
-	if err != nil {
-		return nil, errors.Wrapf(err, "walking workspace %s", workspace)
-	}
-	var dependencies []string
-	for file := range files {
-		dependencies = append(dependencies, file)
+func GetDependencies(ctx context.Context, workspace string, a *latest.CustomArtifact, insecureRegistries map[string]bool) ([]string, error) {
+
+	switch {
+	case a.Dependencies.Dockerfile != nil:
+		dockerfile := a.Dependencies.Dockerfile
+		return docker.GetDependencies(ctx, workspace, dockerfile.Path, dockerfile.BuildArgs, insecureRegistries)
```

Comment:
How is a.Dependencies.Target used?

Reply:
sorry, looks like it isn't, i've removed it